style.css

```diff
@@ -80,4 +80,255 @@
     flex-direction: row;
     list-style-type: none;
   }
+
+  .works {
+    display: flex;
+    flex-direction: column;
+    padding: 2rem;
+  }
+
+  .recent {
+    width: 327px;
+    height: 83px;
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+  }
+
+  .gridcontainer1 {
+    display: grid;
+  }
+
+  .stories {
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    color: #091e42;
+  }
+
+  .selection {
+    font-family: 'Inter', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #091e42;
+  }
+
+  .listbutton {
+    list-style: none;
+    display: flex;
+    gap: 12px;
+    margin-top: 1rem;
+  }
+
+  button:hover {
+    background-color: red;
+  }
+
+  .button1,
+  .button2,
+  .button3,
+  .button4 {
+    border: 1px solid #8993a4;
+    padding: 10px 12px;
+  }
+
+  .button1:hover {
+    background-color: red;
+  }
+
+  .button2:hover {
+    background-color: red;
+  }
+
+  .button3 :hover {
```

Comment:
[ ] kindly change the button hover color to match the provided design 🤓.
The Figma design 👀:
![design](https://user-images.githubusercontent.com/62156607/192270924-0ffcb395-ed27-4fe2-94c0-9d9a3f9b8a01.PNG)

Your design👀  : 
![herdes](https://user-images.githubusercontent.com/62156607/192270968-b0576518-4aab-4238-98f5-be8804d23535.PNG)